lib/core/api.js

```diff
@@ -263,9 +278,10 @@ function postBlockToPlatform(data, groupId, blockIndex, callback) {
 /*
  * process the data sending it to the platform in blocks and feed back progress to the calling function
  */
-api.upload.toPlatform = function(data, sessionInfo, progress, groupId, cb) {
+api.upload.toPlatform = function(data, sessionInfo, progress, groupId, cb, uploadType) {
```

Comment:
As discussed earlier, please just expand on the comment here to explain that the `uploadType` is the final arg (instead of the callback) so that calls to `api.upload.toPlatform` don't have to be modified in _every_ driver...